examples/notedump/Kconfig

```diff
@@ -0,0 +1,30 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+config EXAMPLES_NOTEDUMP
+	tristate "Note dump example"
+	default n
+	depends on SCHED_INSTRUMENTATION_DUMP && NET_UDP
+	---help---
+		Enable the Note dump example
```

Comment:
Maybe the help text can include a short blurb about what the note dump example does? Something like: "This example reads data from the Note driver (/dev/note/ram) and sends it by UDP to the server-addr and server-port specified by command line arguments." WDYT?